otherlibs/stdune-unstable/path.ml

```diff
@@ -610,17 +610,21 @@ module Kind = struct
     | External x -> External (External.relative x (Local.to_string y))
 end
 
-let chmod_generic ~mode ?(op = `Set) path =
+let chmod_generic ~mode ?(stats = None) ?(op = `Set) path =
```

Comment:
Not specific to this PR, but `?(stats = None)` is the same as  `?stats`

Reply:
Oh! Ok then this is this type that's too much: `?stats:Unix.stats option`. Just `?stats:Unix.stats` would do.

Reply:
But, well doesn't matter if we change this to only support setting